worker/src/unstract/worker/clients/helper.py

```diff
@@ -18,5 +17,5 @@ def get_container_client() -> ContainerClientInterface:
         return import_module(client_path).Client
 
     @staticmethod
-    def normalize_container_name(name: str) -> str:
-        return name.replace("/", "-") + "-" + str(uuid.uuid4())
+    def normalize_container_name(name: str, execution_id: str) -> str:
+        return name.replace("/", "-") + "-" + execution_id
```

Comment:
I think one execution will conatins multiple container, each container for each file. So it might be an issue if we add execution Id here

Reply:
@muhammad-ali-e do we have a UUID that uniquely identifies each file's run that's stored or tracked anywhere?

Reply:
@chandrasekharan-zipstack Let's have the container name as `{tool_name}-{execution_id}-container-{short_uuid}`.

Then inside such a container instance, let's prefix the container name to each log output. Try the following:

- Do equivalent of `docker run --name  "{tool_name}-{execution_id}-container-{short_uuid}" alpine:latest`
- Then get value of env var `NAME` in Python code.

Reply:
@hari-kuriakose @muhammad-ali-e @ritwik-g I've updated this PR to provide the below container name
`{tool_name}-{run_id}` - this is to limit the length of the container names and its still traceable by looking up the associated execution_id in the runner / backend logs.
Check the screenshots in the PR for more context

Reply:
@chandrasekharan-zipstack unfortunately ,currently we not other than filehash. But it not suggested for tracing ID.

Reply:
Understood @muhammad-ali-e - we can differentiate that when we bring back multiple tool execution. I've added a comment for this as well. Resolving this to allow merging for now